lnpbp-0001.md

```diff
@@ -363,7 +374,7 @@ pub fn commit(
         .map_err(|_| Error::SumInfiniteResult)?;
 
     let mut hmac_engine =
-        HmacEngine::<sha256::Hash>::new(&pubkey_sum.serialize());
+        HmacEngine::<sha256::Hash>::new(&pubkey_sum.serialize_uncompressed());
```

Comment:
```suggestion
        let key = pubkey_sum.serialize_uncompressed()[1..];
        if key.len() != 64 {
            panic!("scep256k1 key serialization is nonstandard, exiting to prevent validation & commitment errors")
        }
        HmacEngine::<sha256::Hash>::new(key);
```

But in the actual implementation we should check that the length here is 64 bytes